docs/src/modules/java/pages/proto.adoc

```diff
@@ -43,6 +43,19 @@ You define messages that can be sent to or returned from the service. Each messa
 include::example$java-valueentity-shopping-cart/src/main/proto/shoppingcart/shoppingcart_api.proto[tag=proto_messages]
 ----
 
+
+To use a multi-field key, add the `entity_key` notation to each field. For
+example, if the domain proto file `SomeMessage` element defines both
+`first_field` and `second_field` as part of the key, do the same for the service
+message:
```

Comment:
The part about defining entity keys for both domain and service messages doesn't make sense here. Entity keys are only defined on incoming messages for the service, so that messages can be routed to the correct entity.

```suggestion
To use a multi-field key, add the `entity_key` notation to each field. For
example, if the `SomeMessage` service message defines both
`first_field` and `second_field` as part of the key:
```

Reply:
Done.